packages/react/src/alert/Alert.css.ts

```diff
@@ -0,0 +1,116 @@
+import { theme } from "../styles";
+import {
+  type RecipeVariants,
+  createVar,
+  recipe,
+  style,
+} from "../vanilla-extract";
+
+const solidColorVar = createVar();
+const lightColorVar = createVar();
+
+export const alert = recipe({
+  base: [
+    {
+      alignItems: "start",
+      flexDirection: "row",
+      gap: "xs",
+      pl: "md",
+      py: "md",
```

Comment:
any reason not to use just `p` here?
```suggestion
      p: "md",
```

Reply:
didn't use pr here vai for the close icon margin.